php/libraries/NDB_Caller.class.inc

```diff
@@ -141,7 +141,7 @@ class NDB_Caller
             );
 
             $this->instrument = $instrument;
-            header("Location: $redirectToOnSuccess");
+            $redirectToOnSuccess ? header("Location: $redirectToOnSuccess") : '';
```

Comment:
Use if statements, don't abuse the ternary operator (in fact, it's best to avoid using it at all..)